modes/basic-dev-mode/src/index.ts

```diff
@@ -96,7 +96,7 @@ function modeFactory({ modeConfiguration }) {
 
       toolbarService.addButtons(toolbarButtons);
       toolbarService.createButtonSection('primary', [
-        'MeasurementTools',
```

Comment:
I think this should stay MeasurementTools - what it means is that you want to put the "MeasurementTools" button there, which happens to useToolbar('measurementTools'), but that is an internal detail of the MeasurementTools button.